src/k8s/pkg/k8sd/types/cluster_config.go

```diff
@@ -33,12 +33,13 @@ type Network struct {
 }
 
 type Certificates struct {
-	CACert                     string `yaml:"ca-crt,omitempty"`
-	CAKey                      string `yaml:"ca-key,omitempty"`
-	APIServerKubeletClientCert string `yaml:"apiserver-kubelet-client-crt,omitempty"`
-	APIServerKubeletClientKey  string `yaml:"apiserver-kubelet-client-key,omitempty"`
-	K8sDqliteCert              string `yaml:"k8s-dqlite-crt,omitempty"`
-	K8sDqliteKey               string `yaml:"k8s-dqlite-key,omitempty"`
+	CACert                     string   `yaml:"ca-crt,omitempty"`
+	CAKey                      string   `yaml:"ca-key,omitempty"`
+	APIServerKubeletClientCert string   `yaml:"apiserver-kubelet-client-crt,omitempty"`
+	APIServerKubeletClientKey  string   `yaml:"apiserver-kubelet-client-key,omitempty"`
+	K8sDqliteCert              string   `yaml:"k8s-dqlite-crt,omitempty"`
+	K8sDqliteKey               string   `yaml:"k8s-dqlite-key,omitempty"`
+	ExtraSANs                  []string `yaml:"extrasans,omitempty"`
```

Comment:
ExtraSANs are potentially node-level configuration, not part of the cluster config.